index.html

```diff
@@ -9,10 +9,10 @@
     <meta name="description" content="">
     <meta name="author" content="">
 
-    <title>Bare - Start Bootstrap Template</title>
+    <title>Spectra- HAckathon!</title>
 
     <!-- Bootstrap Core CSS -->
-    <link href="css/bootstrap.min.css" rel="stylesheet">
+    <link href="css/bootstrap.css" rel="stylesheet">
```

Comment:
When you move your css into different files, you'll need to import it here.
So something like:
```
<link href="header/frontend/main.css" rel="stylesheet">
<link href="footer/frontend/main.css" rel="stylesheet">
```